target/linux/x86/generic/config-4.19

```diff
@@ -139,6 +139,7 @@ CONFIG_FB_DEFERRED_IO=y
 CONFIG_FB_EFI=y
 CONFIG_FB_HYPERV=y
 # CONFIG_FB_I810 is not set
+CONFIG_FB_SIMPLE=y
```

Comment:
How is this related to this?

Reply:
whitout this config option display is black. it is related to efi display.